x/dsmr/node_test.go

```diff
@@ -1478,6 +1668,120 @@ func getSignerBitSet(t *testing.T, pChain validators.State, nodeIDs ...ids.NodeI
 	return signerBitSet
 }
 
+func Test_Execute(t *testing.T) {
```

Comment:
Looks like we weren't previously testing `Execute`. In hindsight though - I'm not sure how we want to test failure cases for this because I don't think we should be failing `Execute` if `Accept` doesn't return an error (and we shouldn't be calling `Execute` on a block we haven't accepted).

Reply:
Execute should fail if a an invalid signature, expiry, metadata (parent hash, height, timestamp), or duplicate is included.

Accept should execute the internal block, which should be treated as a fatal error, so we can only really test that it correctly propagates a fatal error.